helpdesk_mgmt_team_restrict/security/helpdesk_type_team_restricted.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="helpdesk_team_personal_rule" model="ir.rule">
+            <field name="name">Personal Teams</field>
+            <field ref="helpdesk_mgmt.model_helpdesk_ticket_team" name="model_id" />
+            <field name="domain_force">
+                [
+                    '|',
+                    (user.has_group('helpdesk_mgmt.group_helpdesk_user'), '=', True),
+                    ('id', 'in', user.helpdesk_team_ids.ids)
+                ]
+            </field>
+            <field name="groups" eval="[
+                (4, ref('helpdesk_mgmt.group_helpdesk_user_own')),
+                (4, ref('helpdesk_mgmt.group_helpdesk_user_team'))
```

Comment:
I think it'll work with only the lowest group:

```suggestion
                (4, ref('helpdesk_mgmt.group_helpdesk_user_own')),
```

Because a `group_helpdesk_user_team` user is actually a `group_helpdesk_user_own` user as well.

Could you test?